src/main/java/hlf/java/rest/client/model/NewOrgParamsDTO.java

```diff
@@ -1,13 +1,9 @@
 package hlf.java.rest.client.model;
 
-import java.util.List;
 import lombok.Data;
 
 @Data
-public class NewOrgParamsDTO {
-
+public class NewOrgParamsDTO extends ChannelUpdateParamsDTO {
   private String organizationName;
```

Comment:
[nitpick] don't think this field is being used, can be removed

Reply:
Right, removed. Earlier I didn't remove it thinking the caller would require changes.